docs/user_guide.mdx

```diff
@@ -46,16 +46,15 @@ Once you're ready, click on "continue".
 
 ![](/2024-03-21_user_guide/restore.png)
 
-As long as you have your backup written down somewhere, youre funds will always be safe. **But do not restore your wallet unless it's really needed.
+As long as you have your backup written down somewhere, youre funds will always be safe. \*\*But do not restore your wallet unless it's really needed.
```

Comment:
I guess these changes here were not on purpose?

Reply:
Well they are :)

`dprint fmt` wants it this way.

Reply:
I think this part `\*\*Bu` needs fixing. My assumption is that it misses closing `**` at the end.